Snowflake.Data/Core/IResultChunk.cs

```diff
@@ -2,12 +2,14 @@
  * Copyright (c) 2012-2019 Snowflake Computing Inc. All rights reserved.
  */
 
-using System;
-using System.Collections.Generic;
-using System.Text;
-
 namespace Snowflake.Data.Core
 {
+    public enum ResultFormat
+    {
+        Json,
+        Arrow
+    }
+    
     public interface IResultChunk
     {

```

Comment:
ResultFormat GetResultFormat(); is not required?

Reply:
not yet but it will be